### DISCUSSION TITLE
Responses by environment

### DISCUSSION THREAD

**Discussion by andreshg112** [Kong/insomnia#3712] [Ideas]
- Insomnia Version: Versión 6.3.2 (6.3.2.2588)
- Operating System: macOS Mojave [Version 10.14.2 (18C54)]

## Details

I think it would be good to show response by environment. I have different reponses on them.

For example, this is my Development environment:
![imagen](https://user-images.githubusercontent.com/15859884/51487809-fd518980-1d71-11e9-8ad5-e7b65a1ef697.png)

When I change to Production, I'm still seeing Development response.
![imagen](https://user-images.githubusercontent.com/15859884/51487944-4275bb80-1d72-11e9-8d2c-051ecabeb678.png)

I would like to watch instantly the Production response (only if there is one in the response history).
![imagen](https://user-images.githubusercontent.com/15859884/51487907-325ddc00-1d72-11e9-89f6-d0f46c8141d7.png)

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by gschier** [comment]
This makes a lot of sense and has also come up before for authentication tokens as well
